packages/esm-appointments-app/src/appointments/scheduled/scheduled-appointments.component.tsx

```diff
@@ -69,7 +69,7 @@ const ScheduledAppointments: React.FC<ScheduledAppointmentsProps> = ({ visits, a
     <>
       <ContentSwitcher className={styles.switcher} size="sm" onChange={({ name }) => setCurrentTab(name)}>
         {scheduledAppointmentPanels.filter(shouldShowPanel).map((panel) => {
-          return <Switch key={`panel-${panel.name}`} name={panel.config.title} text={t(panel.config.title)} />;
+          return <Switch key={`panel-${panel.name}`} name={panel.name} text={t(panel.config.title)} />;
```

Comment:
Using `name` (extension name) over `title` as everything else is based on extension name.